docs/resources/waf_policy.md

```diff
@@ -0,0 +1,113 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "netlify_waf_policy Resource - netlify"
+subcategory: ""
+description: |-
+  Netlify Web Application Firewall (WAF) policy. Read more https://docs.netlify.com/security/secure-access-to-sites/web-application-firewall/
+---
+
+# netlify_waf_policy (Resource)
+
+Netlify Web Application Firewall (WAF) policy. [Read more](https://docs.netlify.com/security/secure-access-to-sites/web-application-firewall/)
+
+## Example Usage
+
+```terraform
+resource "netlify_waf_policy" "example" {
+  team_id     = data.netlify_team.team.id
+  name        = "Terraform Ruleset"
+  description = "This is a test ruleset through Terraform"
+  rule_sets = [
+    {
+      managed_id        = "crs-basic",
```

Comment:
is there some const that we can export? I don't know if terraform has those in general

Reply:
I will think about that. Worst-case, maybe a hard-coded data source? I'll let you know what I find.

Reply:
I don't see something in the docs, and ChatGPT also recommended a hard-coded data source - which I don't love. I haven't seen other people doing that.

I will note that there is a static validation on this: https://github.com/netlify/terraform-provider-netlify/pull/77/files#diff-020ba6260ff9fbb79426060f367c74580fbcd04ff8b20c35bbd6345d1acb9157R116

It will fail the `.tf` file before trying to apply, and I think smart IDEs might do a red squiggly if you enter an invalid value.